tests/BookingDataExtracts.UnitTests/BookingDataExtracts.UnitTests.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
I know it's only a small test project but we disabled this in every other csproj so might be good to be consistent?